op-acceptance-tests/tests/sync_tester/sync_tester_elsync/init_test.go

```diff
@@ -10,6 +10,8 @@ import (
 func TestMain(m *testing.M) {
 	presets.DoMain(m,
 		presets.WithExecutionLayerSyncOnVerifiers(),
+		presets.WithMultiWithSyncTester(),
```

Comment:
`presets.WithSimpleWithSyncTester()` is also used together. This sysgo system should be used only once.